pkg/queue/manager.go

```diff
@@ -65,9 +69,23 @@ func (m *Manager) getQueue(deploymentName string) *Queue {
 	defer m.mtx.Unlock()
 	q, ok := m.queues[deploymentName]
 	if !ok {
-		q = New(m.concurrencyPerReplica)
+		q = New(int(m.defaultConcurrencyPerReplica), m.defaultConcurrencyPerReplica)
 		m.queues[deploymentName] = q
 		go q.Start()
 	}
 	return q
 }
+
+func (r *Manager) GetCurrencyPerReplica(deployment string) uint32 {
```

Comment:
typo? should this be `GetConcurrencyPerReplica` instead of `GetCurrencyPerReplica`?